src/test/java/io/confluent/connect/elasticsearch/ElasticsearchClientTest.java

```diff
@@ -565,6 +566,102 @@ public void testNoVersionConflict() throws Exception {
     client2.close();
   }
 
+  @Test
+  public void testIncompatibleESVersionDataStreamNotSet() throws Exception {
+    container.close();
+    container = new ElasticsearchContainer(DEFAULT_DOCKER_IMAGE_NAME + ":" + "7.8.1");
+    container.start();
+
+    String address = container.getConnectionUrl().replace(container.getContainerIpAddress(), container.hostMachineIpAddress());
```

Comment:
```suggestion
    String address = container.getConnectionUrl();
```

should be sufficient, we only do that for the SSL tests